tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -58,7 +59,8 @@ def run(self):
             )
 
         all_ssh_jobs_done = False
-        while not all_ssh_jobs_done:
+        count = 0
```

Comment:
I don't get, how this can be improved. As far as this flow is concerned, It has fired few jobs for writing authorized keys file targeted to other nodes, This flow has to make sure that all these jobs are finished to proceed further. If at all  any of these jobs are not completed after some time, We are clearly reporting the job ids of such jobs.
And in ssh key write job, we will clearly log errors if any that it encountered during writing to ssh keys file. If there are any errors, it means the authorized_key file is not updated, even If we check this we get the same result. All we can do is report the error that ansible encountered while writing the file.

So as far as end user is concerned, when he fires a create/import cluster job, if ssh_copy fails in any of the nodes. He will see a message saying "import failed because following ssh copy jobs failed: <list of ssh jobs that failed>". when he refers to any of those failed job-ids, he will get to know why exactly ssh copy failed.